.nanoclaw/workflows/dev-cycle.sh

```
@@ -0,0 +1,916 @@
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+CONFIG_FILE="${SCRIPT_DIR}/config.env"
+if [[ -f "${CONFIG_FILE}" ]]; then
+  # shellcheck source=/dev/null
+  source "${CONFIG_FILE}"
+fi
+
+# Defaults
+MAX_FIX_ATTEMPTS="${NANOCLAW_WORKFLOW_MAX_FIX_ATTEMPTS:-3}"
+LOG_LEVEL="${NANOCLAW_WORKFLOW_LOG_LEVEL:-info}"
+COLORS="${NANOCLAW_WORKFLOW_COLORS:-auto}"
+GH_REMOTE="${NANOCLAW_WORKFLOW_GH_REMOTE:-origin}"
```

Shellcheck (0.11.0)

[warning] GH_REMOTE appears unused. Verify use (or export if used externally).

(SC2034)